src/containers/HeatMap3DPlotContainer.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import API from '../lib/API.js';
+import HeatMap3DPlot from '../components/charts/heatMap3DPlot/index.jsx';
+
+class HeatMap3DContainer extends React.PureComponent {
```

Comment:
Please rename this Class and filename to something like LargeScaleStructureSOMETHING instead of HeatMapSOMETHING as these are not heat maps, and we may have other components in the future that are.